.github/scripts/update-release-notes.js

```diff
@@ -0,0 +1,160 @@
+        // console.log("statusCode: ", res.statusCode);
+        var data = [];
+        res.on('data', function(chunk) {
+            data.push(chunk);
+        }).on('end', function() {
+            var buffer = Buffer.concat(data);
+            str = new TextDecoder("utf-8").decode(buffer)
+            responseJson = JSON.parse(str)
+            
+            callback(responseJson.published_at, responseJson.body)
+            
+        });
+     
+    });
+     
+    reqGet.end();
+    reqGet.on('error', function(e) {
+        console.error(e);
+    });
+}
+
+
+function updateBOMReleaseNotes() {
+var optionsget = {
+    host : 'search.maven.org', 
+    port : 443,
+    path :  '/solrsearch/select?q=g:com.adobe.marketing.mobile+AND+a:sdk-bom&core=gav&rows=2&wt=json', 
+    method : 'GET',
+    headers: {
+        'Content-Type': 'application/json',
+        'User-Agent': 'server-side',
+      }
+};
+
+var reqGet = https.request(optionsget, function(res) {
+    // console.log("statusCode: ", res.statusCode);
+ 
+    res.on('data', function(d) {
+        str = new TextDecoder("utf-8").decode(d)
+        responseJson = JSON.parse(str)
+
+        responseJson.response.docs.forEach(element => {
+            if (isEarlierThan24Hours(element.timestamp) == false) {
+                if (element.a == 'sdk-bom') {
+                    const version = element.v
+
+                    fetchReleaseNotes('adobe','aepsdk-commons',`bom-${version}`, (ios1086DateStr, releaseNotes) => {
+                        const date = new Date(ios1086DateStr)
+                        const month = date.getMonth()
+                        const day = date.getDate()
+                        const year = date.getFullYear()
+                        const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October","November","December"]
+                        const fullMonth = monthNames[month]
+
+                        const releaseMdPath = "./src/pages/documentation/release-notes/index.md";
+
+                        const releaseNotesHeader = "# Release notes";
+                        // Read the contents of the markdown file.
+                        const contentArray = fs.readFileSync(releaseMdPath, "utf8").toString().split("\n");
+                         // Find the index of the release notes header.
+                        const releaseNotesHeaderIndex = contentArray.indexOf(releaseNotesHeader);
+                        console.error(releaseNotesHeaderIndex)
+
+                        if (releaseFileContainsLineStartWith(`### Android BOM  ${version}`) == true) {
+                            console.error("already updated")
+                            return
+                        }
+                        contentArray.splice(releaseNotesHeaderIndex + 1,0,"",`## ${fullMonth} ${day}, ${year}`,"",`### Android BOM  ${version}`, "",updateBOMReleaseNotesForAdobeIO(releaseNotes));
```

Comment:
This is definitely going to cause issues in the short term when we manually update the release notes for individual releases

Can you also add additional logic
```
if (releaseFileContainsLineStartWith(`### Android BOM  ${version}`) == true) {
    console.error("already updated")
     return
}
let releaseNotesHeaderIndex = contentArray.indexOf(releaseNotesHeader); 
if (releaseFileContainsDate(date)) {
          Adjust the index to be the first line below date
}
```